### ISSUE TITLE
Confirm group name for detekt 2.x artifacts

### ISSUE THREAD

**Issue by 3flex** [detekt/detekt#6054]
Should it be `detekt.dev`? It does tie us to a specific domain name which requires $, but the only realistic alternatives would be `com.github.detekt` or `io.github.detekt` (which could be a contender as it's already used in the package name for some classes etc. in the current plugin).

I'm exploring some backwards compatible changes in the detekt plugin, but they'd introduce new interfaces for public use, so it would be good to align with whatever the artifact group names will be.

**Comment by BraisGabin** [comment]
I would use our domain but the package would be `dev.detekt`. Right?

**Comment by 3flex** [comment]
I've updated the title since for now it's really the package name I'd need, however, conventionally the artifact group name matches the package name prefix, so if we used `dev.detekt` then that would be both the package name prefix as well as the artifact group name.

**Comment by cortinico** [comment]
> Should it be `detekt.dev`?

Yes I'd move everything (package and artifacts) under `dev.detekt.*`.
Are there any counterargument against this?

**Comment by 3flex** [comment]
Only this:

> it does tie us to a specific domain name which requires $

Also someone to maintain. I know this isn't an issue at the moment but I don't know who has access to either the funds or to the domain name registration etc. so I'm not sure what our bus factor is. That's not a problem with `com.github.detekt` or `io.github.detekt`.

Maybe it doesn't matter once the initial publishing is done under the new group name. I'm not familiar with publishing to Maven Central.

**Comment by BraisGabin** [comment]
> Maybe it doesn't matter once the initial publishing is done under the new group name. I'm not familiar with publishing to Maven Central.

It is not. We will not have problems for this.

**Comment by BraisGabin** [comment]
And the idea behind open the donations was to make sure that the founds to keep that founds running is not tied to any of us. The permissions and transfer should be handled but I hope that would not be a problem to maintain that in the future regardless who are the maintainers.

**Comment by cortinico** [comment]
> it does tie us to a specific domain name which requires $

Sorry I missunderstood you mean 'money', I thought `$` was some sort of special identifier.
Once we claimed to central `dev.detekt.`, we can just keep on publishing, even if the domain expires. Anyway, I believe we should make sure the Open Collective has runaway funds to renew the domain as a bare of minimum of 10+ years or so (100/200$).
